config/routes.rb

```diff
@@ -36,7 +36,7 @@
 
     resources :proxy_types,
               path: 'proxy-types',
-              only: [:index]
+              only: %i[index]
```

Comment:
we should disable this lame rubocop rule.  in what world is this stylistically preferable to the line before it?

Reply:
If it is a single it could be either